typescript/libs/editor_client.py

```diff
@@ -86,12 +111,16 @@ def load_format_settings(self):
     def set_features(self):
         host_info = "Sublime Text version " + str(sublime.version())
         # Preferences Settings
-        format_options = {
+        editor_options = {
             "tabSize": self.tab_size,
             "indentSize": self.indent_size,
-            "convertTabsToSpaces": self.translate_tab_to_spaces
+            "convertTabsToSpaces": self.translate_tab_to_spaces,
+            "sendMetrics": self.send_metrics,
+            "telemetryUserID": self.telemetry_user_id,
+            "checkForDtsUpdates": self.check_for_dts_updates
```

Comment:
Should there be some explanation in the `privacyPolicy.txt` that about what `checkForDtsupdates` and other options are for?